<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildingDetailDialog</class>
 <widget class="QWidget" name="BuildingDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0" colspan="3">
    <widget class="QStackedWidget" name="expandStackedWidget">
     <widget class="QWidget" name="machinePage">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="1" column="0">
        <widget class="QLabel" name="machineCurrentProductivity">
         <property name="text">
          <string>MachineCurrentProductivity</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="machineProductLabel">
         <property name="text">
          <string>Product:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="machineMaximumProductivity">
         <property name="text">
          <string>MachineMaximumProductivity</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="selectNextProductComboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QStackedWidget" name="basicOperationStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="notOwnerPage">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="CommandPushButton" name="buyPushButton">
         <property name="text">
          <string>Buy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ownerPage">
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="CommandPushButton" name="sellPushButton">
         <property name="text">
          <string>Sell</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="CommandPushButton" name="dismantlePushButton">
         <property name="text">
          <string>Dismantle</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="detailStackedWidget">
     <property name="minimumSize">
      <size>
       <width>368</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="bankPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="0">
        <widget class="CommandPushButton" name="repayPushButton">
         <property name="text">
          <string>Repay</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="CommandPushButton" name="depositPushButton">
         <property name="text">
          <string>Deposit</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="CommandPushButton" name="openAnAccountPushButton">
         <property name="text">
          <string>Open An Account</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="CommandPushButton" name="withdrawPushButton">
         <property name="text">
          <string>Withdraw</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="CommandPushButton" name="loanPushButton">
         <property name="text">
          <string>Loan</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="CommandPushButton" name="closeAnAccountPushButton">
         <property name="text">
          <string>Close An Account</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="clientStateLabel">
         <property name="text">
          <string>Client State</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="factoryPage">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QPushButton" name="purchaseNewMachineInFactoryPushButton">
         <property name="text">
          <string>Purchase New Machine</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="stopFactoryPushButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="MachineTableWidget" name="factoryTableWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="farmPage">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QPushButton" name="purchaseNewMachineInFarmPushButton">
         <property name="text">
          <string>Purchase New Machine</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="stopFarmPushButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="MachineTableWidget" name="farmTableWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="garagePage">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QPushButton" name="purchaseNewVihiclePushButton">
         <property name="text">
          <string>Purchase New Vihicle</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="stopGaragePushButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="GarageTableWidget" name="garageTableWidget"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="garageStateLabel">
         <property name="text">
          <string>0 / 0 Truck Free</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="minePage">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="purchaseNewMachineInMinePushButton">
         <property name="text">
          <string>Purchase New Machine</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="stopMinePushButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="MachineTableWidget" name="mineTableWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="supermarketPage">
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="1">
        <widget class="QPushButton" name="stopSupermarketPushButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="purchasePushButton">
         <property name="text">
          <string>Purchase</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTableWidget" name="supermarketTableWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="unusedLandPage">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="CommandPushButton" name="buildFactoryPushButton">
         <property name="text">
          <string>Factory</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="CommandPushButton" name="buildBankPushButton">
         <property name="text">
          <string>Bank</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="CommandPushButton" name="buildSupermarketPushButton">
         <property name="text">
          <string>Supermarket</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="CommandPushButton" name="buildFarmPushButton">
         <property name="text">
          <string>Farm</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="CommandPushButton" name="buildGaragePushButton">
         <property name="text">
          <string>Garage</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="CommandPushButton" name="buildVillaPushButton">
         <property name="text">
          <string>Villa</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="CommandPushButton" name="buildMinePushButton">
         <property name="text">
          <string>Mine</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="villaPage"/>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="basicInfoVerticalLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>BuildingName</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="valueLabel">
       <property name="text">
        <string>BuildingValue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>BuildingType</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ownerLabel">
       <property name="text">
        <string>BuildingOwner</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="positionLabel">
       <property name="text">
        <string>BuildingPosition</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GarageTableWidget</class>
   <extends>QTableWidget</extends>
   <header>garagetablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>MachineTableWidget</class>
   <extends>QTableWidget</extends>
   <header>machinetablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>CommandPushButton</class>
   <extends>QPushButton</extends>
   <header>commandpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
